### ISSUE TITLE
Define Unifying Principles for the site

### ISSUE THREAD

**Issue by mgifford** [CivicActions/accessibility#75]
We should look to include some basic principles in building this site:
- We should work to practice what we preach
    - the content should meet the latest WCAG recommendation
    - we need an accessibility statement
    - content should be written in plain language
- We should follow open source best practices
    - content should be modular where possible
    - we shouldn't recreate the wheel. If there is a better resource, just link to it. If it is an open resource, find ways to contribute to it.  If it is on GitHub, fork it.
